### PR TITLE
Catch module import errors in web3ModalConfig

### PR DESCRIPTION
**Pull request by dvinubius**
@ShravanSunder Just tested this successfully, so it does functionally solve the issue.

If the fix at the root is not possible immediately, I guess we're better off with this?

### CODE REVIEW

File: packages/vite-app-ts/src/config/web3ModalConfig.ts [hunk 1]
```diff
@@ -8,66 +8,102 @@ export const web3ModalConfigKeys = {
...
+  try {
+    const Authereum = (await import('authereum')).default;
+    providerOptions.authereum = {
+      package: Authereum,
+    };
+  } catch (e) {
+    console.log('Failed to load config for Authereum: ', e);
+  }
 
+  // === LOCALHOST STATIC
```
Comment (ShravanSunder) [inline packages/vite-app-ts/src/config/web3ModalConfig.ts:104]: should we wrap this in a try catch too?

**Review by ShravanSunder — APPROVED** [review]

File: packages/vite-app-ts/src/config/web3ModalConfig.ts (hunk 1 above)
Comment (dvinubius) [reply packages/vite-app-ts/src/config/web3ModalConfig.ts:104]: Yes, just updated.

My initial thought was that, those two imports being for es6 modules there should be no issue. Atm it works, but it's safer to wrap those two.

And definitely looks prettier in terms of pattern consistency :)

Comment (dvinubius) [reply packages/vite-app-ts/src/config/web3ModalConfig.ts:104]: BTW have I got this right - rollup has issues with commonJS modules, but not with es6 ? 

I mean I looked specifically at the .js files referenced by the imports, and they were ES6 syntax.

Wanna take away as much as possible from this story in terms of learning. Been a wild couple of hours, challenge got me hooked.

Comment (ShravanSunder) [reply packages/vite-app-ts/src/config/web3ModalConfig.ts:104]: in my tests it seems to crash on nodejs utils.  This was shimmed as nodejs modules are not available with browser but lots of web3 libs reference them.  specifically util.inspect.

**Review by ShravanSunder — APPROVED** [review]

File: packages/vite-app-ts/src/config/web3ModalConfig.ts (hunk 1 above)
Comment (ShravanSunder) [reply packages/vite-app-ts/src/config/web3ModalConfig.ts:104]: you can test this locally by running `yarn build --watch`  and `yarn serve --host` to run it in production mode locally

**Pull request merged**